sdk/attestation/azure-security-attestation/pom.xml

```diff
@@ -1,7 +1,9 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
You don't ned the copyright on the pom file.

Reply:
Fair enough - I found another SDK which had one, so I copied it. I'll remove it.